### DISCUSSION TITLE
Booster App Testing

### DISCUSSION THREAD

**Discussion by fecony** [boostercloud/booster#956] [Meta]
The Booster framework does not provide or force users to test their applications. To fix this, I created Issue to make a new CLI command that would create new test files #812. I even started working on this #860.

However, testing in Booster is a difficult task. Developers prefer different libraries for testing, and one library may work better than another. 

In this discussion, I want people to share their experiences and thoughts about Booster app testing, then we can explore which testing library is best for the "Booster way" and stick with it  ✨.

**Comment by fecony** [comment]
###### Discord Conversation started here - https://discord.com/channels/763753198388510780/781960568541806593/897179449458561026

### Possible steps:

- add mocha/chai as dependencies to the project
- create templates for Booster resources
- add `--test -t` cli flag to generate test file with resource
- Integration test example / option? 
- add docs to introduce users to Booster app testing

**Comment by NickSeagull** [comment]
GH Discussions are not being tracked for features, closing.
